packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -44,11 +46,38 @@ export function withViewport<P extends { viewport?: IViewport }, S>(ComposedComp
         });
 
       this._events.on(window, 'resize', this._onAsyncResize);
+
+      this._observer = new MutationObserver((mutations) => {
+        mutations.forEach((mutation) => {
+        if (mutation.type === 'attributes') {
+          if (mutation.attributeName === 'style') {
+            if ((mutation.oldValue.indexOf('display:none') !== -1) || (mutation.oldValue.indexOf('visibility:hidden') !== -1)) {
+              this._updateViewport(true);
+              return;
+            }
+          }
+          if (mutation.attributeName === 'class') {
+            if (mutation.oldValue.indexOf('hide') !== -1) {
```

Comment:
this seems like an unpredictable behavior... we don't always use `is-hidden` or `hide` classes and customers aren't either...